code/modules/admin/verbs/deadsay.dm

```diff
@@ -23,7 +23,7 @@
 	stafftype = "[admin_holder.rank]"
 
 	msg = strip_html(msg)
-	log_admin("[key_name(src)] : [msg]")
+	log_admin("DEAD: [key_name(src)] : [msg]")
```

Comment:
why not just log it to log_say like the rest of dchat logs?

Reply:
Consistency between msay, asay & dsay. Makes it easier to track use of admin tools.

Reply:
Fair enough